### DISCUSSION TITLE
left aligned currency symbol -- right aligned numbers?

### DISCUSSION THREAD

**Discussion by derekwolfson** [django-money/django-money#710] [Q&A] [answered]
Very cool command but is there a way to make it look like this: 

$US &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 1000.93
$US &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 10.45

instead of:

$US 1000.93
&nbsp;&nbsp;&nbsp; $US 10.45

I didn't see it in the documentation -- and this really is a pretty standard way!

If too difficult -- would be nice to have a command to extract the `$US` separately from the value -- so I could just put them in two spans -- currency symbol left aligned and value right aligned.

Thanks -- I'm totally green to Django -- but willing to learn!

**Comment by benjaoming** [comment, marked as answer]
IIRC, you can use `.amount` and `.currency` properties separately on the Money object?

**Reply by derekwolfson** [reply]
Very neat -- I'll give that a shot.
